tsconfig.json

```diff
@@ -31,11 +32,11 @@
   "include": [
     "src/**/*.ts",
     "src/**/*.tsx",
-    "src/**/*.vue",
+    "src/**/*.vue"
+  ],
+  "exclude": [
+    "node_modules",
     "tests/**/*.ts",
     "tests/**/*.tsx"
   ],
-  "exclude": [
-    "node_modules"
-  ]
-}
+}
```

Comment:
missing end of file space

Reply:
I don't know why this would be necessary? Can you maybe elaborate shortly?

Reply:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

there's more where that came from 🤣

Reply:
Thanks for getting back & solved :)